packages/animation-curve-block/src/components/CardText.tsx

```diff
@@ -0,0 +1,61 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { useMemo } from 'react';
+
+import {
+    RichTextEditor,
+    convertToRteValue,
+    getDefaultPluginsWithLinkChooser,
+    joinClassNames,
+} from '@frontify/guideline-blocks-shared';
+import { TextStyles } from '@frontify/fondue';
+
+import { CardTextProps } from '../types';
+import { getTitlePlugin } from '../helpers';
+
+export const CardText = ({
+    appBridge,
+    title,
+    description,
+    hasBorder,
+    designTokens,
+    updateValueOnChange,
+    isEditing,
+    setTitle,
+    setDescription,
+}: CardTextProps) => {
+    const memoTitleRte = useMemo(() => {
+        return (
+            <RichTextEditor
+                value={title || convertToRteValue(TextStyles.ELEMENT_HEADING3)}
+                placeholder="Add a title"
+                onBlur={setTitle}
+                designTokens={designTokens}
+                isEditing={isEditing}
+                updateValueOnChange={updateValueOnChange}
+                plugins={getTitlePlugin()}
+            />
+        );
+    }, [title, designTokens, isEditing, updateValueOnChange, setTitle]);
+
+    const memoDescriptionRte = useMemo(() => {
+        return (
+            <RichTextEditor
+                value={description || convertToRteValue(TextStyles.ELEMENT_PARAGRAPH)}
+                placeholder="Add a description"
+                onBlur={setDescription}
+                designTokens={designTokens}
+                isEditing={isEditing}
+                updateValueOnChange={updateValueOnChange}
+                plugins={getDefaultPluginsWithLinkChooser(appBridge)}
+            />
+        );
+    }, [description, designTokens, isEditing, updateValueOnChange, setDescription, appBridge]);
```

Comment:
isn't that basically all the properties, which would make useMemo obsolete in this case?

Reply:
Its not all properties. 

But i think you are right, do you think it would be better to memoize the whole CardText Component? It was added because of the [comment](https://github.com/Frontify/guideline-blocks/pull/472#issuecomment-1517452414) by @fulopdaniel

Reply:
let's keep it that way if it's performance relevant 👍